src/lhttpc_client.erl

```diff
@@ -159,9 +162,8 @@ execute(From, Host, Port, Ssl, Path, Method, Hdrs0, Body, Options) ->
         requester = From,
         request_headers = Hdrs,
         socket = Socket,
-        connect_timeout = proplists:get_value(connect_timeout, Options,
-            infinity),
-        connect_options = proplists:get_value(connect_options, Options, []),
+        connect_timeout = proplists:get_value(connect_timeout, Options, DefConTimeout),
+        connect_options = proplists:get_value(connect_options, Options, DefConOptions),
```

Comment:
`connect_options` is a list. so for me it looks natural to merge `DefConOptions` and `connect_options` from `Options`, but not drop all `DefConOptions` in case any option is passed to `execute()` call.
current version will increase entropy of the universe.

Reply:
fair enough. agree. will make it soon

Reply:
+1 here.